calcom/cal.com
# feat: Added Drag and Drop for Event Types

## What does this PR do?



This PR adds support for Drag and Drop in the Event Types list. The PR adds `@dnd-kit/core`, `@dnd-kit/modifiers`, `@dnd-kit/sortables` and `@dnd-kit/utilities`, which are widely used packages for handling Drag and Drop.

Fixes #10016
/claim #10016


[https://www.loom.com/share/7da594acde784c86858b8cf756616c1c?sid=d28c278a-c189-48ea-a6ee-1ddab6367444](https://www.loom.com/share/7da594acde784c86858b8cf756616c1c?sid=d28c278a-c189-48ea-a6ee-1ddab6367444)

## Type of change




## How should this be tested?



The change is primarily a UI functionality change. To test the change, you can drag around the list items on `/event-types`. Refresh the page to confirm that items have moved places. Also test that the original buttons still work.

## Mandatory Tasks


## Checklist



- I haven't checked if my PR needs changes to the documentation
- I haven't added tests that prove my fix is effective or that my feature works
